content/swift/concepts/dictionaries/terms/contains/contains.md

```diff
@@ -1,47 +1,50 @@
 ---
-Title: '.contains()'
-Description: 'Returns a Boolean value indicating whether the sequence contains the given element.'
+Title: 'contains'
+Description: 'A Boolean value that indicates whether a dictionary contains a specified key.'
```

Comment:
Description should always start with a verb. Please rewrite it.

Reply:
@Radhika-okhade 

Committed changes on this. Thanks much